app/controllers/profiles_controller.rb

```diff
@@ -4,10 +4,18 @@ class ProfilesController < ApplicationController
   def index
     @location = params[:location]
     @radius = 20
+    @interests = Interest.all
+    @selected_interests = params[:selected_interests]
     @profiles = return_profiles_based_on_location(@location, @radius)
+    if !@selected_interests.nil?
```

Comment:
Is there a reason for the duble negative: if !@selected_interests.nil?

could you consider having the following, to make it even easier to read? - or am I missing something?

if @selected_interests.nil?
      @profiles_by_interest = @profiles
    else
      @profiles_by_interest = return_profiles_based_on_interests(@profiles)
    end

Reply:
you obviously does not have to change if you do not agree - as it works as it is :D

Reply:
I was just too lazy to switch it out so i put an exclamation mark in front, gonna fix that now :p